script.js

```diff
@@ -1,4 +1,66 @@
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  // 1 = Scissors, 2 = Paper, 3 = Stone
+  var randomAction = randomSPS();
+  myOutputValue =
+    'Error. Please input only "Scissors", "Paper" or "Stone". Try agin!';
```

Comment:
As this is the first time you are declaring myOutputValue you'll need to use var